lib/lite_cable/iodine.rb

```diff
@@ -0,0 +1,179 @@
+# coding: utf-8
+# frozen_string_literal: true
+
+# TODO:
+# 1. broadcast работает только изнутри сервера Iodine(т.е. если хочется его из экшенов
+# синатры, а не из канала, то надо синатру и каналы запускать сервером iodine).
+# Заработает снаружи, если сделать что-то из этого:
+# a) сделать http путь типо /broadcast изнутри iodine rack.
+# b) запустить и синатру и каналы через сервер iodine(т.е. пуму убрать)
```

Comment:
I think, we need b) and c).

Let's by default consider that we're running within Iodine (btw, is it possible to check whether we're inside Iodine or not?)

In order to send broadcasts from other processes, one should explicitly _enable_ Redis broadcast.

Reply:
The issue in Iodine is solved, so b) is works. 

We can check that Redis is used as pub/sub engine, but I couldn't find the method for check if we're inside Iodine - I'll ask Iodine creator about this.

Also, for Redis broadcast from outside Iodine, it need to add Redis gem - in Litecable or in user's app, because Iodine itself uses C implementation and don't have Redis dependency. 

It's okay if I'll add Redis in Litecable deps, or it's better to add a comment in Readme?

Reply:
It's better to add optional dependencies as runtime ones. So, let's add a note about it to Readme and a check in the code like this:

```ruby
gem "redis", ">= 3.0.0"
```

That would automatically raise an appropriate error if gem is not loaded (or loaded with a wrong version)